roles/common/tasks/get_authorization_token.yml

```diff
@@ -41,7 +41,7 @@
     force_basic_auth: true
   register: auth_token_mds
   until: auth_token_mds.status == 200
-  retries: 10
+  retries: "{{ mds_retries }}"
```

Comment:
Was this any customer reported issue or was a miss from our end?

Reply:
This works for certain cases and for certain other it needs more retries. But all our tests used to run with 10 retries. But yesterday some customer reported that for them it takes more retries. Also in 7.8 I had noticed the same thing and changed the behaviour to use a variable here to adjust the number of retries